typescript/sdk/src/token/EvmERC20WarpModule.hardhat-test.ts

```diff
@@ -33,13 +33,22 @@ import { ProxyFactoryFactories } from '../deploy/contracts.js';
 import { HyperlaneIsmFactory } from '../ism/HyperlaneIsmFactory.js';
 import { MultiProvider } from '../providers/MultiProvider.js';
 import { AnnotatedEV5Transaction } from '../providers/ProviderType.js';
+import { RemoteRouter } from '../router/types.js';
 import { randomAddress } from '../test/testUtils.js';
 import { ChainMap } from '../types.js';
 
 import { EvmERC20WarpModule } from './EvmERC20WarpModule.js';
 import { TokenType } from './config.js';
 import { TokenRouterConfig } from './schemas.js';
 
+const randomRemoteRouters = (n: number) => {
+  const routers: RemoteRouter = {};
+  for (let domain = 0; domain < n; domain++) {
+    routers[domain] = randomAddress();
```

Comment:
just sanity-checking here that `domain` is always just an integer index? What else might domain be?

Reply:
domain/domainId is always an integer, it's chainId that can be either an integer or string